pyspark/test/resources/conf/test_spark-bigdl.conf

```diff
@@ -27,16 +27,6 @@
 # https://github.com/intel-analytics/BigDL/wiki/Programming-Guide#engine
 #
 
-spark.executorEnv.DL_ENGINE_TYPE                    mklblas
```

Comment:
It would be great to be able to remove this, but doesn't the MKL .so need to read these env variables? if there's another way to do it that's great